### PR TITLE
Special character issue

### PR DESCRIPTION
**Pull request by caitlynachen**
This PR is aimed at addressing issue #175.

### CODE REVIEW

File: lux/vislib/altair/BarChart.py [hunk 1]
```diff
@@ -40,44 +40,55 @@ def initialize_chart(self):
...
 
+        x_attr_abv = x_attr.attribute
+        y_attr_abv = y_attr.attribute
+
+        if len(x_attr.attribute) > 25:
+            x_attr_abv = x_attr.attribute[:15] + "..." + x_attr.attribute[-10:]
+        if len(y_attr.attribute) > 25:
+            y_attr_abv = y_attr.attribute[:15] + "..." + y_attr.attribute[-10:]
+
+        x_attr.attribute = x_attr.attribute.replace(".", "")
```
Comment (dorisjlee) [inline lux/vislib/altair/BarChart.py:51]: Why is the replace "." done once in AltairRenderer.py and once again inside each of the chart type? Is this necessary?

File: tests/test_columns.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+
+def test_abbrev_histogram():
+    dataset = [
+        {long_var: 1},
+        {long_var: 0},
+    ]
+    test = pd.DataFrame(dataset)
+    vis = Vis([long_var], test).to_Altair()
+    "axis=alt.Axis(labelOverlap=True, title='Lorem ipsu...aborum. (binned)')" in vis
```
Comment (dorisjlee) [inline tests/test_columns.py]: Failing test here, missing agg_title case

**Review by dorisjlee — CHANGES_REQUESTED** [review]

**Comment by dorisjlee** [comment]
The assert for the last three test cases was missing, so I added them back in. It seems like `test_abbrev_agg` was not covered and needs to be patched.
image

**Comment by caitlynachen** [comment]
hmm... for me the test case you mentioned was failing was working on my end! lmk if it's still an issue
Screen Shot 2020-12-26 at 9 37 04 AM

File: lux/vislib/altair/BarChart.py (hunk 1 above)
Comment (caitlynachen) [reply lux/vislib/altair/BarChart.py:51]: this is renaming the actual object attribute parameter. i believe altair Renderer changes the column names but does not change the x_attr.attribute

**Review by dorisjlee — APPROVED** [review]

**Comment by dorisjlee** [comment]
This looks great, thanks @caitlynachen!

**Pull request merged**